ui/apps/platform/src/Containers/MainPage/Sidebar/NavigationSidebar.tsx

```diff
@@ -36,103 +46,63 @@ const platformConfigurationPaths = [
-                    title={basePathToLabelMap[violationsBasePath]}
-                />
-                <LeftNavItem
-                    isActive={location.pathname.includes(complianceBasePath)}
-                    path={complianceBasePath}
-                    title={basePathToLabelMap[complianceBasePath]}
-                />
-                <NavExpandable
-                    id="VulnerabilityManagement"
-                    title="Vulnerability Management"
-                    isActive={vulnerabilityManagementPaths.some((path) =>
-                        location.pathname.includes(path)
-                    )}
-                    isExpanded={vulnerabilityManagementPaths.some((path) =>
-                        location.pathname.includes(path)
-                    )}
-                >
-                    {vulnerabilityManagementPaths.map((path) => {
-                        const isActive =
-                            path === vulnManagementPath ? false : location.pathname.includes(path);
-                        return (
-                            <LeftNavItem
-                                key={path}
-                                isActive={isActive}
-                                path={path}
-                                title={basePathToLabelMap[path]}
-                            />
-                        );
-                    })}
-                </NavExpandable>
-                <LeftNavItem
-                    isActive={location.pathname.includes(configManagementPath)}
-                    path={configManagementPath}
-                    title={basePathToLabelMap[configManagementPath]}
-                />
-                <LeftNavItem
-                    isActive={location.pathname.includes(riskBasePath)}
-                    path={riskBasePath}
-                    title={basePathToLabelMap[riskBasePath]}
-                />
-                <NavExpandable
-                    id="PlatformConfiguration"
-                    title="Platform Configuration"
-                    isActive={platformConfigurationPaths.some((path) =>
-                        location.pathname.includes(path)
-                    )}
-                    isExpanded={platformConfigurationPaths.some((path) =>
-                        location.pathname.includes(path)
-                    )}
-                >
-                    {platformConfigurationPaths.map((path) => {
-                        const isActive = location.pathname.includes(path);
-                        return (
-                            <LeftNavItem
-                                key={path}
-                                isActive={isActive}
-                                path={path}
-                                title={basePathToLabelMap[path]}
-                            />
-                        );
-                    })}
-                </NavExpandable>
+                {[dashboardPath, networkBasePath, violationsBasePath, complianceBasePath]
```

Comment:
Sometimes I've found it a bit hard to visually parse something like this vs. creating a variable for it and defining it further above in the component. Maybe it would be helpful here